content/management/configuration-api/v3.3/index.mdx

```diff
@@ -888,18 +888,23 @@ curl -X POST \
 
 ### List Bots
 
+Lists all the Bots within a license that were created by the requester. The method behavior differs depending on who created the Bot and who the requester is.
```

Comment:
I think `...that were created by the requester` is misleading but I don' know how to rephrase it.
This method returns all bots created via given application if `all` is set to `false` and all bots on the license otherwise, but it's not possible to get all bots created by the requester.

Reply:
Hm, my idea was to represent either an agent or an application as a requester, but it's not very intuitive. Maybe, we should make the first sentence more general:

Returns the list of Bots created within a license.

Reply:
👏 sounds good